From 3377fe12b628b0f5c571fe3bfb8f9cf34d0a8fc4 Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bart@iconica.nl>
Date: Tue, 10 May 2022 11:37:52 +0200
Subject: [PATCH] Changed dependency on pyusb to accept anything higher than
 1.0.0

It can't be expected that everybody will use 1.0.0 only, e.g.
distribution packaging. I've already confirmed it works fine with pyusb
1.2.0 as well, so let's just accept anything higher than 1.0.0
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 6166e89..910f22b 100644
--- a/setup.py
+++ b/setup.py
@@ -31,7 +31,7 @@ def read(fname):
     ]
 else:
     os_requires = [
-        "pyusb==1.0.0"
+        "pyusb>=1.0.0"
     ]
 
 setup(
